spot-client/src/spot-remote/ui/components/dial-pad/StatelessDialPad.js

```diff
@@ -136,10 +137,12 @@ export default class StatelessDialPad extends React.Component {
                         <div className = 'dial-pad-footer-button'>
                             <button
                                 className = { callButtonClassName }
-                                disabled = { this.props.disableCallButton }
+                                disabled = { this.props.disableCallButton || this.props.dialingInProgress }
                                 onClick = { this._onSubmit }
                                 type = 'submit'>
-                                <Call />
+                                { this.props.dialingInProgress
+                                    ? <div className = 'dialing-dots' >...</div>
```

Comment:
Is it not possible to use the loading-icon component? Does it not look okay?

Reply:
I will try that